src/store/slices/counterSlice.ts

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+type counterState = {
```

Comment:
type name does not follow naming convention
```suggestion
type CounterState = {
```